.github/workflows/slowtest.yaml

```diff
@@ -47,7 +47,7 @@ jobs:
         run: just nextest --features embedded-db --no-run
 
       - name: Run slow test
-        run: just nextest --features embedded-db --no-fail-fast --nocapture
+        run: RUST_LOG=off just nextest --features embedded-db --no-fail-fast --nocapture
```

Comment:
Remove `--nocapture` instead?

Reply:
done in [ac3644d](https://github.com/EspressoSystems/espresso-network/pull/2706/commits/ac3644d9740673668f46034164f9d77162e18b53)